httpbin/core.py

```diff
@@ -649,6 +648,91 @@ def links(n):
     return redirect(url_for('link_page', n=n, offset=0))
 
 
+@app.route('/file/<path:p>', methods=('GET', 'POST',))
```

Comment:
thoughts on decoupling this method into two seperate methods? one for get and one for post. methods should ideally be doing one and only specified job really well.  and possibly move the dup code into a helper method that performs the path stuff.